consultation_analyser/consultations/dummy_data.py

```diff
@@ -151,5 +215,27 @@ def create_dummy_consultation_from_yaml(
                         ThemeMappingFactory(
                             answer=answer, theme=theme, execution_run=theme_mapping_run
                         )
+                    logger.info(
+                        f"Added theme mappings for question number {question.number} and question part number {question_part.number}"
+                    )
+                    if include_changes_to_themes:
+                        simulate_user_amending_themes(question_part)
 
     return consultation
+
+
+# Will only be run occasionally to create dummy data
+# Not in prod
+@job("default", timeout=1800)
```

Comment:
think this timeout might need to be 2100 to work on my machine (though happy to leave it if you think dev will be fast enough - can always update the timeout locally on the rare occasion I run this task)

Reply:
Will update to be 2100.